app/models/nufs_account.rb

```diff
@@ -1,5 +1,6 @@
 class NufsAccount < Account
-  validates_uniqueness_of :account_number, :message => "already exists"
+  validates_uniqueness_of :account_number, message: "already exists"
+  validates_presence_of :expires_at
```

Comment:
There was a handful of validations taking place in deprecated controller methods.  Such as:
```
account.valid? # populate virtual charstring attributes required by set_expires_at
account.errors.clear

# be verbose with failures. Too many tasks (#29563, #31873) need it
begin
  account.set_expires_at!
  account.errors.add(:base, I18n.t('controllers.facility_accounts.create.expires_at_missing')) unless account.expires_at
rescue AccountNumberFormatError => e
  account.expires_at = Time.zone.now # Prevent expires_at missing message
rescue ValidatorError => e
  account.errors.add(:base, e.message)
end
```